policy/leaving-governance.md

```diff
@@ -4,13 +4,24 @@ This documents the process of leaving Electron Governance. A contributor is no l
 
 ## Offboarding Items
 
-* Update README file for the Working Group. Typically the current WG chair will submit this, but any member of the WG can submit.
+1. A governance member (Typically the WG Chair) update README file in [governance](https://github.com/electron/governance) for the WG the member is leaving.
 
-* Submit a PR in [.permissions repo](https://github.com/electron/.permissions/). Typically the current WG chair will submit this, but any member of the WG can submit.
-    * Merging this PR will automatically remove the contributor from:
-        * The WG's team on GitHub
-        * The WG's user group on Slack
+2. A governance member (Typically the WG Chair) submits a PR in [.permissions repo](https://github.com/electron/.permissions/).
+  * Merging this PR will automatically remove the contributor from:
+    * The WG's team on GitHub
+    * The WG's user group on Slack
 
-* Request an admin of Electron's Slack to change the contributor to a guest account. Message in #wg-community-safety for this. Any member of Community & Safety WG is a Slack admin.
+3. Reset the former member's Slack account email to a personal email.
 
-* Message in #access-request that the contributor has left Electron Governance. The Google suite account needs to be manually disabled, and 30 days later the account will be deleted. See [details on GSuite permissions](./permissions.md#gsuite).
+4. A Slack Admin demotes the member's account from Full Member to Multi-Channel Guest
```

Comment:
Steps 4 onward do not apply if the contributor is a member of another working group